app/scripts/modules/core/src/pipeline/config/triggers/artifacts/docker/defaultDocker.artifact.ts

```diff
@@ -0,0 +1,62 @@
+import { module } from 'angular';
+
+import { PIPELINE_CONFIG_PROVIDER } from 'core/pipeline/config/pipelineConfigProvider';
+import { IArtifact } from 'core/domain/IArtifact';
+import { PipelineConfigProvider } from 'core/pipeline';
+import { isNullOrUndefined } from 'util';
```

Comment:
where does this come from?  i dont see `util` in package.json and can't find a file or function with isNullOrUndefined as its name.

lodash has a function for this - `isNil`

Reply:
I don't know, but it seems to work - @danielpeach does this make sense?

Reply:
switching out with `isNil` for now

Reply:
intellij says it comes from node. babel probably has an implementation that it writes into the build if you use it.